new/LIT7416QeneLazeMaya.xml

```diff
@@ -34,8 +34,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>The author of the qǝne is not indicated in the manuscript. The type of the qǝne is indicated as 
-                    <foreign xml:lang="gez">ሥላሴ</foreign>.</p>
+                <p>The author of the qǝne is not indicated in the manuscript. The type of the qǝne is indicated as śǝllāse.</p>
```

Comment:
Sorry, but when transliterated it should still be encoded with `<foreign xml:lang="gez"/>`.